samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,393 @@
+
+.. note::
+
+   The Radio Domain manifest template is only created when building the Bluetooth Low Energy version of the sample, and not the UART version.
+   Currently, it is not needed for the UART version.
+
+If you want to make modifications to how the DFU is executed in this sample, you can do so by editing the manifest templates.
+See the :ref:`ug_nrf54h20_suit_customize_dfu` user guide for instructions and examples.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/suit/smp_transfer`
+
+This sample can be found under |sample path| in the |NCS| folder structure.
+
+.. caution::
+   In specific instances, after a cold boot, the nRF54H20 DK can behave like a bricked device (being unreachable via debugger, or non-booting).
+   To fix the issue, briefly tap (for less than 0.5 seconds) the Reset button on the DK.
+
+Building and programming using the command line
+===============================================
+
+To build and program the sample to the nRF54H20 DK, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Over Bluetooth Low Energy
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample using the following ``west`` command, with the following Kconfig options set:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp -- -DOVERLAY_CONFIG=overlay-bt.conf -DCONFIG_APP_VERSION=1
+
+         .. note::
+
+            If you are compiling in Windows and the build is unsuccessful due to the maximum path length limitation, use the following command:
+
+            .. code-block:: console
+
+               west build -p -b nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir -- -DOVERLAY_CONFIG=overlay-bt.conf -DCONFIG_APP_VERSION=1
+
+         If you want to further configure your sample, see :ref:`configure_application` for additional information.
+
+         After running the ``west build`` command, the output build files can be found in the :file:`build/zephyr` directory.
+         The following build artifacts are found with both firmware binaries embedded as integrated payloads:
+
+          * :file:`root.suit` - This file is the most important envelope, containing embedded :file:`app.suit` and :file:`hci_rpmsg_subimage.suit` as an integrated dependencies.
+            These files are used to execute DFU.
+          * :file:`app.suit` - This file is the envelope for the Application Core, containing embedded Application Core firmware as an integrated payload.
+          * :file:`hci_rpmsg_subimage.suit` - This file is the envelop for the Radio Core, containing embedded radio core firmware as an integrated payload.
+
+         For more information about files generated as output of the build process, see :ref:`app_build_output_files`.
+         For more information on the contents of the build directory, see :ref:`zephyr:build-directory-contents` in the Zephyr documentation.
```

Comment:
If there is a specific page that show the file structure generated by sysbuild build, it will be worth to use it instead (there are some changes, i.e. application artifacts are under a dedicated subfolder).

Reply:
Good point! Will look into this and get back to this.

Reply:
The link you provided looks different than the one you implemented, @kittydepa , so I'm not sure if this line has been updated. Anyway, I'm not sure if providing a link to output files in the NCS with the detailed tables first, and then linking to a generic section in Zephyr with way less detailed list makes sense.